### ISSUE TITLE
Provide a possibility to execute jbang online

### ISSUE THREAD

**Issue by OLibutzki** [jbangdev/jbang#287]
Having a online tool which is aware of executing jbang scripts would be very beneficial. 

A browser extension which adds an Execute button next to a GitHub Gist would be nice as well (similar to what [Gitpod](https://www.gitpod.io/docs/browser-extension/) does).

**Comment by maxandersen** [comment]
yeah, i've pinged repl.it guys if interested in providing a more cleaner out of the box experience but they don't see to be that keen on java stuff.

I do have gitpod setup in jbang itself; see i.e. https://github.com/jbangdev/jbang/blob/2a174896b9209283c8ea89a267864170d2982dcf/.gitpod.Dockerfile ...but its not really tuned for jbang simplicity. ideas welcome :)

**Comment by OLibutzki** [comment]
Any new opportunities to execute jbang in a browser?

**Comment by maxandersen** [comment]
Shell.azure.com
Gitpod
GitHub codespaces
Eclipse Che
Repl.it
